lib/mongo/movies.js

```diff
@@ -0,0 +1,43 @@
+const { default: clientPromise } = require("@/lib/mongo/db");
+
+let client;
+let db;
+let moviesCollection;
+
+async function init() {
```

Comment:
What do you think about moving from functional to class based structure? Then we can have constructors and we can init the db there.